### PR TITLE
Add netl-ap-map-flow

### PR DESCRIPTION
**Pull request by stadelmanma**
First go at this, time to see if I got the Fortran dependencies correct. 
Main repo is at: https://github.com/stadelmanma/netl-ap-map-flow

### CODE REVIEW

**Comment by stadelmanma** [comment]
Any advice on getting my Fortran code to compile? The model in this package isn't intimately linked into the Python code, instead the Python code just calls the final binary produced from the Fortran source. So the Python code itself builds just fine. 

For reference here are the two errors that popped up during Circle:
```
/opt/rh/devtoolset-2/root/usr/bin/ld: cannot find -lm
/opt/rh/devtoolset-2/root/usr/bin/ld: cannot find -lm
/opt/rh/devtoolset-2/root/usr/bin/ld: cannot find -lc
collect2: error: ld returned 1 exit status
```
 and Travis CI:
```
ld: library not found for -lcrt0.o
```

**Comment by stadelmanma** [comment]
@conda-forge/core, @jakirkham  I have managed to get my package to compile on all platforms, however I have a few questions to make sure I have done everything correctly, and with best practices in mind.

Questions:
- Is there a better way to access 64bit MinGW in AppVeyor? `set PATH=%PATH%;C:\mingw-w64\x86_64-6.3.0-posix-seh-rt_v5-rev1\mingw64\bin` <- this seems inherently fragile (in bld.bat)

- I had initially assumed when compiling binary code that we wanted statically linked files to avoid issues with dynamic lib dependences. However I can only get the Windows build to compile statically. Do we want static binaries? Or will the conda packages always satisfy the libgfortran dependency upon `conda install ...`

In terms of clarifying how this package works, the Fortran code is small and vanilla with no dependencies other than the Fortran runtime itself. Additionally, the Fortran code is currently tangential to the model with no intimate link between the two via f2py, i.e. the Fortran model can run as a standalone program. The python module serves as a high level interface to run the model, pre-process data, post-process data, export data int other formats etc.

File: recipes/netl-ap-map-flow/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ repo_name }}-{{ version }}.tar.gz
+  url: {{ repo_base_url }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k]
```
Comment (mariusvniekerk) [inline recipes/netl-ap-map-flow/meta.yaml]: This line is probably redundant

File: recipes/netl-ap-map-flow/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+{% set version = "0.0.2" %}
+{% set sha256 = "0c571ac190fe0a69aa94a654f37dc778b487704d4b491144b875e500adc296a6" %}
+{% set py_mod_deps = ['numpy x.x', 'scipy', 'pillow >=3.4.0', 'pyyaml']%}
+
+package:
+  name: {{ repo_name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ repo_name }}-{{ version }}.tar.gz
```
Comment (mariusvniekerk) [inline recipes/netl-ap-map-flow/meta.yaml]: Is there a more stable source than this for tarball?  Github does occasionally recreate these which can cause hash failures.

**Comment by stadelmanma** [comment]
@mariusvniekerk I could pull the zip file at [Zenodo instead](https://zenodo.org/record/495729#.WPDzIHnD-JB) if zip files are acceptable.

 If there is a more stable github integration I can use for tarballs I'll be glad to add that in but I don't intend to host the code on PyPI since the Fortran code makes things awkward.

File: recipes/netl-ap-map-flow/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    - python
+    - setuptools
+    - toolchain
+    - libtool  # [not win]
+    - gcc  # [not win]
+    - libgfortran  # [not win]
+    - m2w64-gcc-fortran  # [win]
+    {% for each_py_mod_dep in py_mod_deps %}
+    - {{ each_py_mod_dep }}
+    {% endfor %}
```
Comment (isuruf) [inline recipes/netl-ap-map-flow/meta.yaml]: Can you add these explicitly like the others without the loop? conda-forge's maintenance scripts won't be able to function properly with this.

File: recipes/netl-ap-map-flow/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  skip: True  # [linux32]
+  skip: True  # [win32]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - libtool  # [not win]
+    - gcc  # [not win]
```
Comment (isuruf) [inline recipes/netl-ap-map-flow/meta.yaml]: `# [osx]` is enough

File: recipes/netl-ap-map-flow/meta.yaml (hunk 2 above)
Comment (isuruf) [inline recipes/netl-ap-map-flow/meta.yaml]: I didn't know Github would recreate these. Have they mentioned this somewhere?

Comment (stadelmanma) [inline recipes/netl-ap-map-flow/meta.yaml]: I know my tarball was rebuilt and the checksum changed when I renamed my repo but I suppose that would be expected.

**Comment by stadelmanma** [comment]
@isuruf @mariusvniekerk I have made the requested changes to the meta.yaml file. Not sure what to do about the tarball source if that is a common issue with github release tags.

**Comment by stadelmanma** [comment]
@isuruf is there anything I need to do to get this merge-ready?

**Comment by stadelmanma** [comment]
This project will likely not be actively developed anymore beginning next month and instead simply maintained as dependencies change over time. So I am closing this PR and will instead just host it on my own channel.

**Pull request closed without merging**